insights/parsers/getconf_pagesize.py

```diff
@@ -0,0 +1,32 @@
+"""
+getconf PAGE_SIZE - command ``/usr/sbin/getconf PAGE_SIZE``
+===========================================================
+
+This very simple parser returns the output of the ``getconf PAGE_SIZE`` command.
+
+Examples:
+
+    >>> pagesize_parsed.page_size
+    '4096'
+"""
+from . import ParseException
+from .. import parser, CommandParser
+from insights.specs import Specs
+
+
+@parser(Specs.getconf_page_size)
+class GetconfPageSize(CommandParser):
+    """Class for parsing 'getconf PAGE_SIZE' command output
+
+    Output: page_size
+    """
+
+    def parse_content(self, content):
+        if len(content) != 1:
+            msg = "getconf PAGE_SIZE output contains multiple non-empty lines"
+            raise ParseException(msg)
+        raw = content[0].strip()
+        self.page_size = raw
```

Comment:
Would it make sense to go ahead and convert this to an integer value here so that developers don't have to perform this conversion in their rules?